vendor/brave-ios/Shared/RewardsLogStream.h

```diff
@@ -6,19 +6,29 @@
 
 #import <Foundation/Foundation.h>
 #import <iostream>
+#import <ostream>
 
 #import "bat/ledger/ledger_client.h"
 #import "bat/ads/ads_client.h"
 #import "Logger.h"
 
+class LogStream {
+ public:
+  virtual ~LogStream() = default;
+  virtual std::ostream& stream() = 0;
+};
+
 /// A generic unbuffered logger logs messages via iostream
-class RewardsLogStream : public ledger::LogStream, public ads::LogStream {
+class RewardsLogStream : public ledger::LogStream, public LogStream {
 public:
   /// Creates a stream for logging ledger information
   RewardsLogStream(const char* file, const int line, const ledger::LogLevel log_level);
     
   /// Creates a stream for logging ads information
-  RewardsLogStream(const char* file, const int line, const ads::LogLevel log_level);
+  RewardsLogStream(const char* file, const int line, const ads::LogSeverity severity);
+    
+  /// Creates a stream for logging verbose ads information
+  RewardsLogStream(const char* file, const int line, const int verbose_level);
```

Comment:
How are we supposed to differentiate `verbose_level`? We use the `BATLogLevel` enum to determine log severity in iOS: https://github.com/brave/brave-ios/blob/d7cad1fb49de908a9b280101a9178757be8fcff0/Client/Frontend/Browser/BrowserViewController/BVC%2BRewards.swift#L30

Reply:
Also why do we need 2 different loggers? Couldn't we just include a boolean for `is_verbose` or something to the log method?

Reply:
> We have two kinds of logging. `LOG(severity, ...)` with severities of `INFO`, `WARNING` and `ERROR`. ` and `VLOG(verbose_level, ...)` where the verbose level is 0, 1, 2, 3... see vendor/bat-native-ads/src/bat/ads/internal/logging.h comments

Reply:
@tmancey Any suggestions on what we do then? Considering we aren't using Chromium logging system and iOS has their own logging system already in place. 

Should we just turn all `VLOG` into debug logs? Should we map out the integer based verbosity to info, warning, etc.?

Reply:
At this time lets just log all VLOG/LOG into debug logs